Libplanet.Unity/SwarmRunner.cs

```diff
@@ -37,6 +37,11 @@ public class SwarmRunner
             _cancellationTokenSource = new CancellationTokenSource();
         }
 
+        /// <summary>
+        /// A flag for whether the swarmPreloadTask has completed.
+        /// </summary>
+        public bool PreloadTask_ended { get; set; }
```

Comment:
```suggestion
        public bool isPreloaded { get; set; }
```
looks good to remove `_`